deploy/resource-quotas/15-selectorsyncset.clusterresourcequota.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: hive.openshift.io/v1alpha1
```

Comment:
Think this files should be deleted.

Reply:
Indeed. "Longdayitus" - I forgot `git add 10* 15*` won't add the deleted files to the changeset.